Makefile

```diff
@@ -0,0 +1,14 @@
+# Building docker image
+VERSION = "v1.0.0"
+IM = shalsh/phenoscape-pipeline-tools
```

Comment:
Can you please put in the phenoscape org? https://hub.docker.com/u/phenoscape 
Do you have the right privileges?

Reply:
I don't think so. It says only owners can create repositories. Maybe I can publish if the repository is already there?

Reply:
@hlapp do you know?

Reply:
That's possible. What should the repo name be? `phenoscape-pipeline-tools` or simply `pipeline-tools`? I assume the latter?

Reply:
I like `pipeline-tools`.

Reply:
I created that now, @balhoff. Note that you're an owner of the organization, so in theory you should have been able to do this too. Did you get an error or some other trouble?

Reply:
I went to the site and it didn't look like I had the ability to create new repositories. I will look again.